06-Education_training.Rmd

```diff
@@ -302,6 +302,13 @@ to compensate for the time required to participate. In addition, to increase par
 from a diverse community, URSSI will provide travel support for a small number of workshop 
 participants, determined based on need in individual cases. 
 
+6. _Packaging and Sharing_
+As URSSI develops content related to the educational activities described in this chapter, we
+will package that content into Carpentries-style lessons. We will make the content freely 
+available on GitHub. We will also archive each release of content on Zenodo using a proper 
+license that allows anyone to use and/or modify the content as needed. Finally, we will seek
+to work with the Carpentries to have URSSI content included as Carpentries content.
```

Comment:
Are we paying lip service to this or do we really plan to do this? The carpentries cant really take this on without a fte (otherwise we're offering them a free puppy that they have to feed, walk and raise). In the past Library Carpentry came about because of an IMLS grant that pad for a 3 year FTE to coordinate the effort. Other proposed carpentries haven't come together because of lack of proper resources. So unless we're budgeting for this and have a MOU in the works, we should not commit them to doing anything.

Reply:
I removed the last sentence in the item.

Reply:
We can add it later after a discussion with the Carpentries leadership if needed, but for now we shouldn't without an explicit discussion with them.

Reply:
Sounds good. Can I merge this change now or do you have other thoughts?

Reply:
good to merge.